src/main/kotlin/net/atos/zac/signalering/SignaleringZACHelper.kt

```diff
@@ -13,7 +13,7 @@ import net.atos.zac.flowable.FlowableTaskService
 import org.flowable.task.api.TaskInfo
 import java.util.UUID
 
-class SignaleringenZACHelper @Inject constructor(
+class SignaleringZACHelper @Inject constructor(
```

Comment:
Naming question: should this not be `SignaleringZacHelper` ?

Reply:
Hmm, maybe. I don't think we made a decision on our preference regarding capitals yet. Also, the code base is very full of capital only usages. E.g. all the `REST` services..

Reply:
I do agree however so will rename it while I'm at renaming.. :-)

Reply:
I'll leave it for now. It is quite a strange service anyway I think. It is not a help at all, but rather a mini-service used by the SignaleringService only. The word `ZAC` also does not add anything here really. I'll make a separate PR for it. :-)